test/library/packages/BLAS/COMPOPTS

```diff
@@ -1 +1,2 @@
 -L $CRAY_LD_LIBRARY_PATH -g -lgfortran -lsci_gnu
+# -I /opt/homebrew/Cellar/openblas/0.3.27/include/ -L /opt/homebrew/Cellar/openblas/0.3.27/lib/ -lopenblas
```

Comment:
I have the same problem locally, I renamed `COMPOPTS` -> `COMPOPTS.ignore` and invoke `LD_LIBRARY_PATH=$HOME/openblas/opt/lib start_test -compopts"-L$HOME/openblas [...]"`

Maybe this is required for CI tests ?

Reply:
Hi @ninivert — The existing line is because we currently are only testing this directory on Crays, where BLAS is easily available to us and in a well-defined location.  The "right" thing to do here would be to have the BLAS.notest in the previous directory check for whether BLAS is available, regardless of the system or its installed location, and to rewrite this file to be a script that would automatically add any necessary `-I`, `-L`, `-l` lines to use it.  But I'm not really sure how to do that in a portable manner, so for now, I just captured the line that happens to work on my Mac today, where I manually swap between these lines to test things out locally.  Not ideal, but better than not being able to test locally at all...